packages/loopring_v3/contracts/core/impl/ExchangeV3.sol

```diff
@@ -586,6 +586,21 @@ contract ExchangeV3 is IExchangeV3, ReentrancyGuard
         }
     }
 
+    function approveTransactionsWithoutEvents(
```

Comment:
Maybe change `approveTransactions` to take an extra boolean argument to log events or not? Or if we don't want to change the existing API use a shared internal function that takes a boolean so the code doesn't need to be duplicated. 

Or just remove the event from `approveTransactions` because it's extremely likely the only variant that will be used any way.

Reply:
Sounds reasonable to me. I prefer just to remove the event from approveTransactions.